src/test/java/org/dataone/speedbagit/ProfilingTest.java

```diff
@@ -0,0 +1,119 @@
+package org.dataone.speedbagit;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.BufferedOutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Random;
+import java.util.zip.ZipOutputStream;
+
+/**
+ * A suite of tests that should be run under a profiler and ignored by CI systems and ordinary builds.
+ * These should be run before each release to ensure that emory management is
+ * sane (ie entire files aren't loaded into memory at once).
+ */
+
+public class ProfilingTest {
```

Comment:
In these two tests, it seems like we should be asserting that the bags contain the correct files, and perhaps that it is created in an expected timeframe, although that might be hard to estimate.  Anyway, these unit tests are effectively manually verified until there's some sort of success assertion.

Reply:
I left the validation tests out of this file since the machinery for validating the generated bags is in the SpeedBagIt test file. If something is going wrong with the content of the generated bags it should be caught in those tests and we should see the failure.